cards/card_component.js

```diff
@@ -26,6 +31,34 @@ BaseCard["{{componentName}}"] = class extends ANSWERS.Component {
         })
       );
     }
+
+    let feedbackFormSelector = '.js-HitchhikerCard-feedbackForm';
```

Comment:
We typically use `const` instead of `let` when the value doesn't change. There are a few spots in this pr where we can use `const` instead of let